### DISCUSSION TITLE
Need to disable masked storage in database

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#364] [Sunbird ED configuration discussion] [answered]
Hello All,

The database currently stores the value in the masked form .
Is there any way to disable it such that the database will store the umasked data directly.

![maskeddata](https://user-images.githubusercontent.com/65657373/213621100-7fd34d60-aeda-4ee6-97b9-812172f8a57f.png)

Please guide me here .

cc : @coolbung

**Comment by reshmi-nair** [comment, marked as answer]
@Xitija DB currently stores the data in encrypted form and masked form in 2 columns in sunbird.user table. In case of phone  number, it is phone and maskedphone. By default encryption is ON for PII data. You can update the sunbird_encryption=ON property to OFF in environment variable list or in externalresource.properties file for switching off encryption. Then it will get stored in plain text.

**Reply by rhwarrier** [reply]
Please note that phone and email are considered PII data, and it is recommended to keep them encrypted for storage. If you're going to use plain text, there may be a need to keep your users informed/ take appropriate actions as policy recommends. Please take all considerations into account before a decision is taken, and the config changes are made for the SB instance. 
cc :  @SanthoshVasabhaktula
